scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+## parameters
+CONTRACT_ADDRESS="link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8"
+FROM_ACCOUNT="link146asaycmtydq45kxc8evntqfgepagygelel00h"
+URL="http://localhost:26658"
+CODE_ID=1
+
+# Wait until the height of the block is greater than or equal to 1.
+timeout 60 bash -c "until fnsad query block 1 --node ${URL} --chain-id simd-testing --keyring-backend test; do sleep 0.5 done"
+
+# If the timeout fails, the process will be terminated abnormally.
+exitstatus=$?
+if [[ $exitstatus -ne 0 ]]; then
+    echo "timeout failed"
+    exit 1
+fi
+
+# send transaction to node
+cur_path=`pwd`
+
+artifacts_path="artifacts"
+
+# This is a function that checks if the result is as expected.
+check_result() {
+    if [[ "$result" == *"failed"* ]]; then
+      exit 1
+    fi
+}
+
+#*** store collection contract ***
+result=$(fnsad tx wasm store ${artifacts_path}/collection.wasm --node ${URL} --from ${FROM_ACCOUNT} --chain-id simd-testing --keyring-backend test --gas 10000000 -b block -o json -y)
```

Comment:
`--chain-id simd-testing --keyring-backend test -b block -o json -y` 
You can set these parameters as one const, and use it easily.

Reply:
Fixed as pointed out.
https://github.com/Finschia/finschia-wasm/pull/3/commits/0780d9f07c9811cafcf7332984792c038d1b63ee